auth/src/main/java/com/firebase/ui/auth/ui/email/RegisterEmailFragment.java

```diff
@@ -226,11 +233,11 @@ private void validateAndRegisterUser() {
         boolean nameValid = mNameValidator.validate(name);
         if (emailValid && passwordValid && nameValid) {
             mHandler.startSignIn(new IdpResponse.Builder(
-                    new User.Builder(EmailAuthProvider.PROVIDER_ID, email)
-                            .setName(name)
-                            .setPhotoUri(mUser.getPhotoUri())
-                            .build())
-                    .build(),
+                            new User.Builder(EmailAuthProvider.PROVIDER_ID, email)
```

Comment:
Is something wrong with the formatting settings?

Reply:
I wouldn't worry about this.  Equally legible either way.

Reply:
Nope, we just made an indentation change to the styling a long time ago and never bothered running it over the whole codebase. Like @samtstern said, we're not _that_ nit picky. 😆 Thanks for checking though! 👍

I'll take another look at this tomorrow.